experimental/generation/generator/templates/professional/professional.schema

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "$templates": [
+        "AboutBot.qna",
+        "AboutUser.qna",
+        "Command.qna",
+        "Compliment.qna",
+        "Criticism.qna",
+        "Dialog.qna",
+        "Greeting.qna",
+        "Relationship.qna",
+        "library-CrossTrain.lg",
```

Comment:
You can get rid of your qna function if you add a locale.qna.lg file that is like the locale.lg.lg or locale.lu.lg file in standard.